xmtp_proto/src/gen/xmtp.message_api.v1.tonic.rs

```diff
@@ -1,5 +1,6 @@
 // @generated
 /// Generated client implementations.
+#[cfg(feature = "tonic")]
 pub mod message_api_client {
```

Comment:
this is the effect of that ^ buf config change on the generated output (i.e. exclude it unless `tonic` feature is enabled)

Reply:
(the `xmtp_proto/Cargo.toml` already had the `grpc` feature configured to imply the `tonic` feature)